### PR TITLE
SGSSB: Add VXN's Custom ability

### PR DESCRIPTION
**Pull request by ghost**
Everyone has their fun, why can't I

**K READY TO MERGE**

### CODE REVIEW

File: mods/sgssb/moves.js [hunk 1]
```diff
@@ -600,7 +600,35 @@ exports.BattleMovedex = {
 			this.add('raw|<div class=\"broadcast-red\"><b>The server has crashed:</b><br/>TypeError: Cannot read property \'Overpowered\' of undefined at CommandContext.meme (./SpacialGaze/spacialgaze-plugins/hoeenhero/spellcheck.js:420:69)</div>');
 		},
 		onPrepareHit: function (target, source) {
-			this.add('-anim', source, "Hex", source);
+			this.add('-anim', source, "10000000 Volt Thunderbolt", target);
```
Comment (HoeenCoder) [inline mods/sgssb/moves.js:603]: This is ALOT of animations, too many tbh

Comment (DeathlyPlays) [reply mods/sgssb/moves.js:603]: But it isn't illegal to xD

Comment (HoeenCoder) [reply mods/sgssb/moves.js:603]: True, im just concerned about lag ect

Comment (DeathlyPlays) [reply mods/sgssb/moves.js:603]: Its every Z Move animation kek

**Pull request merged**